eos-tech-support/eos-install-firefox

```diff
@@ -0,0 +1,341 @@
+#!/bin/bash -e
+
+# Downloads Firefox and unzips to the user's home directory,
+# with a desktop launcher
+
+exit_with_error() {
+   echo $1
+   exit 1
+}
+
+USERID=$(id -u)
+if [ "$USERID" == "0" ]; then
+    exit_with_error "Program cannot be run as superuser"
+fi
+
+DEB_ARCH=`dpkg --print-architecture`
+case ${DEB_ARCH} in
+    amd64) ;; # Only 64-bit Intel architectures are supported
+    *) exit_with_error "Unsupported architecture ${DEB_ARCH}"
+       ;;
+esac
+
+TEMP_DIR=`mktemp -d`
+DEST_DIR=~/.firefox
+WGET_COMMAND="wget -c --tries=5"
+DOWNLOAD_DIR=`xdg-user-dir DOWNLOAD`
+
+FIREFOX_PATTERN=${DOWNLOAD_DIR}/firefox-*.tar.bz2
+FIREFOX_URL=https://www.mozilla.org/firefox/new/?scene=2
+
+JAVA_PATTERN=${DOWNLOAD_DIR}/jre-*.tar.gz
+JAVA_URL=http://javadl.oracle.com/webapps/download/AutoDL?BundleId=216424
+
+# While the following would be a faster download of the current version,
+# previous versions are often removed from the server, so it is safest
+# for an automated script to use the stable version in the archive
+# http://fpdownload.macromedia.com/get/flashplayer/pdc/11.2.202.643/install_flash_player_11_linux.x86_64.tar.gz
+FLASH_VERSION=11.2.202.643
+FLASH_SHA256SUM="1a79f38f7153d436083b5a6b5d0f38e0c4cd345e83e7d050152452057f83350a"
+FLASH_FILENAME=fp_${FLASH_VERSION}_archive.zip
+FLASH_URL=http://download.macromedia.com/pub/flashplayer/installers/archive/${FLASH_FILENAME}
+
+download_via_browser() {
```

Comment:
I don't see any reason to download this via browser. A stable download link for the latest version can always be made by following these instructions: https://download-installer.cdn.mozilla.net/pub/firefox/releases/latest/README.txt